extensions/blocks/podcast-player/components/header.js

```diff
@@ -1,67 +1,95 @@
+
+/**
+ * WordPress dependencies
+ */
 import { memo } from '@wordpress/element';
 
 const Header = memo(
-	( { playerId, title, cover, link, track, children, showCoverArt, showEpisodeDescription } ) => (
-		<div className="jetpack-podcast-player__header-wrapper">
-			<div className="jetpack-podcast-player__header" aria-live="polite">
-				{ showCoverArt && cover ? (
-					<div className="jetpack-podcast-player__track-image-wrapper">
+	( { playerId, title, cover, link, track, children, showCoverArt, showEpisodeDescription, colors } ) => (
+		<div className="jetpack-podcast-player__header">
+			<div className="jetpack-podcast-player__current-track-info">
+				{ showCoverArt && cover && (
+					<div className="jetpack-podcast-player__cover">
 						{ /* alt="" will prevent the src from being announced. Ideally we'd have a cover.alt, but we can't get that from the RSS */ }
-						<img className="jetpack-podcast-player__track-image" src={ cover } alt="" />
+						<img className="jetpack-podcast-player__cover-image" src={ cover } alt="" />
 					</div>
-				) : null }
+				) }
 
-				{ title || ( track && track.title ) ? (
-					<div className="jetpack-podcast-player__titles">
-						<Title playerId={ playerId } title={ title } link={ link } track={ track } />
-					</div>
-				) : null }
+				{ !! ( title || ( track && track.title ) ) && (
+					<Title
+						playerId={ playerId }
+						title={ title }
+						link={ link }
+						track={ track }
+						colors={ colors }
+					/>
+				) }
 			</div>
 
 			{ /* putting this above the audio player for source order HTML with screen readers, then visually switching it with the audio player via flex */ }
-			{ showEpisodeDescription && track && track.description ? (
+			{ !! ( showEpisodeDescription && track && track.description ) && (
 				<div
 					id={ `${ playerId }__track-description` }
 					className="jetpack-podcast-player__track-description"
 				>
 					{ track.description }
 				</div>
-			) : null }
+			) }
 
 			{ /* children contains the audio player */ }
 			{ children }
 		</div>
 	)
 );
 
-const Title = memo( ( { playerId, title, link, track } ) => (
-	<h2 id={ `${ playerId }__title` } className="jetpack-podcast-player__titles">
-		{ track && track.title ? (
-			<span className="jetpack-podcast-player__track-title">{ track.title }</span>
-		) : null }
+const Title = memo( ( {
+	playerId,
+	title,
+	link,
+	track,
+	colors = { primary: { name: null, custom: null, classes: '' } }
+} ) => (
+	<h2 id={ `${ playerId }__title` } className="jetpack-podcast-player__title">
+		{ !! ( track && track.title ) && (
+			<span
+				className={ classnames( 'jetpack-podcast-player__current-track-title', colors.primary.classes ) }
+				style={ { color: colors.primary.custom } }
+			>
+				{ track.title }
+			</span>
+		) }
 
 		{ /* Adds a visually hidden dash when both a track and a podcast titles are present */ }
-		{ track && track.title && title ? (
+		{ !! ( track && track.title && title ) && (
```

Comment:
@WunderBart @marekhrabe I think I remember these `!!` in the if statements being removed in one of your PRs. Do you recall?

Reply:
I actually added them in https://github.com/Automattic/jetpack/pull/15211/commits/16cd880e67d9f3667b4e2c1fe27475f97d07bef7, because of https://github.com/Automattic/jetpack/pull/15211#issuecomment-606663384 :)